napari/_vispy/_tests/test_vispy_labels_layer.py

```diff
@@ -95,3 +99,35 @@ def test_labels_painting_with_mouse(
     assert not np.any(visual.node._data)
     mouse_press_callbacks(layer, event)
     assert np.any(visual.node._data)
+
+
+@skip_local_popups
+@skip_on_win_ci
+def test_labels_iso_gradient_modes(qtbot):
+    """Check that we can set `iso_gradient_mode` with `iso_categorical` rendering (test shader)."""
+    # NOTE: this test currently segfaults on Windows CI, but confirmed working locally
+    # because it's a segfault, we have to skip instead of xfail
```

Comment:
Still fail on Windows?

Reply:
Sadly yes, see this run
https://github.com/napari/napari/actions/runs/10172415519/job/28135289123

Reply:
ok. Looks like OpenGL error (happens in `vispy\gloo\glir`). Maybe worth adding information that it is rendered related segfault.